platform/security/handler/security-handler-pki/src/main/java/org/codice/ddf/security/handler/pki/CrlChecker.java

```diff
@@ -108,20 +111,37 @@ public void setCrlLocation(String location) {
     /**
      * Generates a new CRL object from the given location.
      *
-     * @param location Path to the CRL file
-     * @return A CRL object constructed from the given file path. Null if an error occurred while attempting to read the file.
+     * @param location File Path or URL to the CRL file
+     * @return A CRL object constructed from the given file path or URL. Null if an error occurred while attempting to read the file.
      */
     private CRL createCrl(String location) {
-        try (FileInputStream fis = new FileInputStream(new File(location))) {
+        try {
```

Comment:
I believe you still want to handle this with a Try with resources like it was before. The try/catch could have an issue with closing the input stream on line 127.